.travis/travis_utils.sh

```diff
@@ -157,3 +164,89 @@ bell() {
     sleep 60
   done
 }
+
+create_multiarch_manifest_planet(){
+    build_message Creating Planet Multiarch Manifests
+    if [ "$BRANCH" = "master" ]
```

Comment:
can you try temporarily to run this on `multiarch_suport` branch and push to `planet-test`?

Reply:
Yes, will do that for testing